docs/Installing-Bouncy-Castle.md

```diff
@@ -1,13 +1,18 @@
 # Installing Bouncy Castle
 
-In order to function correctly, PowerAuth software requires Bouncy Castle to be available. While some servers, such as Wildfly, already come with own BC version, other servers, for example Tomcat, requires BC to be installed system-wide. You also need to install BC for testing the server via our command-line utility.
+In order to function correctly, PowerAuth software requires Bouncy Castle to be available.
 
-## Java 8
+Bouncy Castle library installation depends on Java version and used web container.
```

Comment:
I like the detailed documentation, but I would try to structure it a bit better so that I can find my config faster. Also, I would put Java 11 first.

# Installing on Java 11
## Configuring Java Security
## Bouncy Castle on Tomcat
## Bouncy Castle on Wildfly
## Testing the installation

Reply:
Done